cypress-E2E-tests/cypress/support/dtm_generic.commands.ts

```diff
@@ -301,8 +300,7 @@ Cypress.Commands.add('resetTheColumnFilter', columnName => {
   cy.wait('@reset', { timeout: 10000 })
     .its('response.statusCode')
     .should('eq', 200);
-  //added wait to resolve flakyness
-  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  // added wait to resolve flakyness
```

Comment:
Add a description of what kind of flakiness is observed when the wait is not present

Reply:
those are unused commands and I have removed.